app/models/manageiq/providers/openstack/infra_manager/host.rb

```diff
@@ -41,6 +41,10 @@ def cloud_tenant
   def security_groups
   end
 
+  def self.network_port_type
```

Comment:
B/c generic method written in models/Host.rb expects the network_port_type. The "Super method" in Host.rb was replaced by ImplementMe Exception.